hbase-balancer/src/test/java/org/apache/hadoop/hbase/master/balancer/BalancerTestBase.java

```diff
@@ -285,6 +285,9 @@ protected List<ServerAndLoad> convertToList(final Map<ServerName, List<RegionInf
   }
 
   protected String printMock(List<ServerAndLoad> balancedCluster) {
+    if (balancedCluster == null) {
+      return "null";
+    }
```

Comment:
A null list here likely means a test failure, but you're likely to find a more clear failure message somewhere other than here